bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/api/WebexTeamsApi.java

```diff
@@ -128,30 +123,29 @@ private <I, O> O doRequest(URI url, HttpMethod method, String authToken, Class<O
                 req.header("Content-type", "application/json");
             }
 
-            InputStreamResponseListener listener = new InputStreamResponseListener();
-            req.send(listener);
+            ContentResponse response = req.send();
 
-            // Wait for the response headers to arrive
-            Response response = listener.get(5, TimeUnit.SECONDS);
```

Comment:
Please note that you can set a specific timeout to your request using `req/timeout(xxx)`. If not, the default timeout will be considered, meaning 30s I believe. I don't know what you want.

Reply:
The default will be just fine.  So I'll leave it as is.